frontend/src/component/playground/Playground/AdvancedPlayground.tsx

```diff
@@ -28,6 +28,56 @@ const StyledAlert = styled(Alert)(({ theme }) => ({
     marginBottom: theme.spacing(3),
 }));
 
+const GenerateWarningMessages: React.FC<{
+    response?: AdvancedPlaygroundResponseSchema;
+}> = ({ response }) => {
+    const invalidContextProperties =
+        response?.warnings?.invalidContextProperties;
+
+    if (invalidContextProperties && invalidContextProperties.length > 0) {
+        invalidContextProperties.sort;
```

Comment:
Should this be `invalidContextProperties.sort()`?

Reply:
Yes... yes it should. Good point. I .. should probably test that it does that too 😅

Reply:
Done in 74026919882db9a6ab0fd9e05854b905b8a4989d